src/formats/PDB.cpp

```diff
@@ -215,31 +240,30 @@ void PDBFormat::read_HELIX(string_view line) {
         return;
     }
 
-    auto chain1 = line[19];
-    auto chain2 = line[31];
-    int64_t start = 0;
-    int64_t end = 0;
-    auto inscode1 = line[25];
-    auto inscode2 = line[37];
+    auto chain_start = line[19];
+    auto chain_end = line[31];
+    auto inscode_start = line[25];
+    auto inscode_end = line[37];
+    auto resname_start = trim(line.substr(15, 3)).to_string();
+    auto resname_end = trim(line.substr(27, 3)).to_string();
```

Comment:
So this property is in the HELIX records?

Reply:
Yes it is also repeated there: http://www.wwpdb.org/documentation/file-format-content/format33/sect5.html#HELIX